src/main/api/ledger/thorchain/transaction.ts

```diff
@@ -116,7 +129,106 @@ export const sendTx = async ({
     // Send signed StdTx
     const {
       data: { txhash }
-    } = await auth.txsPost(sdk, stdTx, 'block')
+    } = await auth.txsPost(cosmosClient.sdk, stdTx, 'block')
+
+    if (!txhash) {
+      return E.left(LedgerErrorId.SEND_TX_FAILED)
+    }
+
+    return E.right(txhash)
+  } catch (error) {
+    return E.left(getErrorId(error))
```

Comment:
After the merge with #1749 this will need to be updated because of the new TS version